### PR TITLE
[OpenVINO-EP 2021.4] Update Ubuntu Dockerfile w.r.t 2021.4

### PR DESCRIPTION
**Pull request by MaajidKhan**
**Description**: Update Ubuntu Dockerfile w.r.t 2021.4

**Motivation and Context**
updated code for ubuntu dockerfile w.r.t 2021.4
Added unattended-upgrade package to deploy stage along with it's dependencies
VADM Fix available as part of this changes

### CODE REVIEW

File: dockerfiles/Dockerfile.openvino [hunk 1]
```diff
@@ -1,92 +1,61 @@
...
+
+# Deploy stage
+FROM openvino/ubuntu18_runtime:${OPENVINO_VERSION}
+
+ENV DEBIAN_FRONTEND noninteractive
+
+USER root
+COPY --from=builder /home/openvino/onnxruntime/build/Linux/Release/dist/*.whl ./
+COPY --from=builder /GPL_sources.tar.gz ./
+RUN python3 -m pip install ./*.whl && rm ./*.whl
```
Comment (smkarlap) [inline dockerfiles/Dockerfile.openvino:49]: Pip install should be done as USER and not as root. Installing as root will install the package into the root’s python environment and may not be accessible or modify able when running the image as user.  
move this line and preferably the two preceding COPY lines to after the ‘USER ${BUILD_USER}’ line.

File: dockerfiles/README.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ Use `docker pull` with any of the images and tags below to pull an image and try
 |-------------------|---------------------------------------|---------------------------------------|-------------------------|
 | Source (CPU)      | mcr.microsoft.com/azureml/onnxruntime | :v0.4.0, :v0.5.0, v0.5.1, :v1.0.0, :v1.2.0, :v1.3.0, :v1.4.0, :v1.5.2 | :latest |
 | CUDA (GPU)        | mcr.microsoft.com/azureml/onnxruntime | :v0.4.0-cuda10.0-cudnn7, :v0.5.0-cuda10.1-cudnn7, :v0.5.1-cuda10.1-cudnn7, :v1.0.0-cuda10.1-cudnn7, :v1.2.0-cuda10.1-cudnn7, :v1.3.0-cuda10.1-cudnn7, :v1.4.0-cuda10.1-cudnn7, :v1.5.2-cuda10.2-cudnn8 | :latest-cuda            |
-| OpenVino          | mcr.microsoft.com/azureml/onnxruntime | :v1.6.0-openvino-2021.1.110, :v1.7.0-openvino-2021.2.200 | :latest-openvino |
+| OpenVino          | hub.docker.com/repository/docker/openvino/onnxruntime_ep_ubuntu18 | :2021.3 | :latest |
```
Comment (smkarlap) [inline dockerfiles/README.md:27]: The tag should be 2021.4 now

File: dockerfiles/README.md [hunk 3]
```diff
@@ -113,20 +111,19 @@ If the *device_type* runtime config option is not explicitly specified, CPU will
   | <code>GPU_FP32</code> |Intel<sup></sup> Integrated Graphics |
   | <code>GPU_FP16</code> | Intel<sup></sup> Integrated Graphics |
   | <code>MYRIAD_FP16</code> | Intel<sup></sup> Movidius<sup>TM</sup> USB sticks |
-  | <code>VAD-M_FP16</code> | Intel<sup></sup> Vision Accelerator Design based on Movidius<sup>TM</sup> MyriadX VPUs |
```
Comment (smkarlap) [inline dockerfiles/README.md:116]: VAD-M is working now, right? So why is this being removed?

File: dockerfiles/README.md [hunk 4]
```diff
@@ -113,20 +111,19 @@ If the *device_type* runtime config option is not explicitly specified, CPU will
...
   | <code>HETERO:<DEVICE_TYPE_1>,<DEVICE_TYPE_2>,<DEVICE_TYPE_3>...</code> | All Intel<sup>®</sup> silicons mentioned above |
   | <code>MULTI:<DEVICE_TYPE_1>,<DEVICE_TYPE_2>,<DEVICE_TYPE_3>...</code> | All Intel<sup>®</sup> silicons mentioned above | 
 
   Specifying Hardware Target for HETERO or Multi-Device Build:
 
   HETERO:<DEVICE_TYPE_1>,<DEVICE_TYPE_2>..
   MULTI:<DEVICE_TYPE_1>,<DEVICE_TYPE_2>..
-  The <DEVICE_TYPE> can be any of these devices from this list ['CPU','GPU','MYRIAD','FPGA','HDDL']
+  The <DEVICE_TYPE> can be any of these devices from this list ['CPU','GPU','MYRIAD']
```
Comment (smkarlap) [inline dockerfiles/README.md:121]: Should no longer remove vad-m documentation.

File: dockerfiles/README.md [hunk 5]
```diff
@@ -113,20 +111,19 @@ If the *device_type* runtime config option is not explicitly specified, CPU will
...
 
   A minimum of two DEVICE_TYPE'S should be specified for a valid HETERO or Multi-Device Build.
 
   Example:
-  HETERO:MYRIAD,CPU  HETERO:HDDL,GPU,CPU  MULTI:MYRIAD,GPU,CPU
+  HETERO:MYRIAD,CPU  HETERO:GPU,CPU  MULTI:MYRIAD,GPU,CPU
```
Comment (smkarlap) [inline dockerfiles/README.md:126]: Should no longer remove vad-m documentation.

File: dockerfiles/README.md [hunk 6]
```diff
@@ -166,34 +163,6 @@ If the *device_type* runtime config option is not explicitly specified, CPU will
     docker run -it --rm --device-cgroup-rule='c 189:* rmw' -v /dev/bus/usb:/dev/bus/usb onnxruntime-myriad:latest
 
     ```
-
-### OpenVINO on VAD-M Accelerator Version
```
Comment (smkarlap) [inline dockerfiles/README.md:170]: Should no longer remove vad-m documentation.

File: dockerfiles/README.md (hunk 5 above)
Comment (MaajidKhan) [reply dockerfiles/README.md]: Added back

**Comment by MaajidKhan** [comment]
This was a draft PR for Dockerfiles task. closing this as we have created a actual PR with Msft.

**Msft PR:** https://github.com/microsoft/onnxruntime/pull/8491/

**Pull request closed without merging**